src/language-css/loc.js

```diff
@@ -69,18 +69,25 @@ function calculateValueNodeLoc(node, rootOffset, text) {
 }
 
 function getValueRootOffset(node) {
-  return (
-    node.source.startOffset +
-    (typeof node.prop === "string" ? node.prop.length : 0) +
-    (node.type === "css-atrule" && typeof node.name === "string"
-      ? 1 + node.name.length + getLeadingWhitespaceLength(node.raws.afterName)
-      : 0) +
-    (node.type !== "css-atrule" &&
+  let result = node.source.startOffset;
+  if (typeof node.prop === "string") {
+    result += node.prop.length;
+  }
+
+  if (node.type === "css-atrule" && typeof node.name === "string") {
+    result +=
+      1 + node.name.length + node.raws.afterName.match(/^[\s:]*/)[0].length;
```

Comment:
Did you mean `/^[\s:]?/` ？

Reply:
No. It calculates the length of the leading substring that consists of only colons and whitespace.
E.g. `\t\t:\t//foo` → 4.

Reply:
I added this explanation as a comment in the code.

Reply:
Sorry, my main concern is about `:`, shouldn't only  one colon be  matched? So `/^(?:\s*:\s*)?/`?

Reply:
If you know a way how more than one colon can end up there, then yes.

Reply:
I don't know, but if you mean to match one `:`, I think you should write in this way.